### PR TITLE
Remote control for spacenav

### PR DESCRIPTION
**Pull request by kiselev-dv**
## Test scenario:

make ssh connection with port 6564 forward onto displaynode. Check that port is open in iptables rules.

run ros node on display node:

    rosrun spacenav_remote server.py

run client script

    scripts/client.py

Now you should be able to control camera on the remote LG

### CODE REVIEW

File: spacenav_remote/package.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+  <author email="dmitry@endpoint.com">Dmitry Kiselev</author>
+  <maintainer email="lg@endpoint.com">End Point Liquid Galaxy Team</maintainer>
+
+  <license>TODO</license>
+
+  <url type="website">https://github.com/EndPointCorp/lg_ros_nodes</url>
+
+  <buildtool_depend>catkin</buildtool_depend>
+  <build_depend>rospy</build_depend>
```
Comment (wzin) [inline spacenav_remote/package.xml]: missing `spnav` from here

File: spacenav_remote/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+## Remote control for spacenav
+
+### Usage
+
+1. Run spcenav_remote ros node on displaynode
+2. Open the port 6564 on the remote system by adding the below to /etc/iptables.conf:
+`-A INPUT -p tcp -s 10.42.0.0/16 --dport 6564 -j ACCEPT`
+3. Forward port `6564`
+4. Install the `spnav` package: `pip install spnav`
```
Comment (wzin) [inline spacenav_remote/README.md]: this can be automaticaly done via Cmakelist.txt - please move it there

File: spacenav_remote/package.xml (hunk 1 above)
Comment (kiselev-dv) [inline spacenav_remote/package.xml]: It's server, it doesn't use `spnav` spnav used by client.

File: spacenav_remote/README.md (hunk 2 above)
Comment (kiselev-dv) [inline spacenav_remote/README.md]: I puted port forwarding into live build, `pip install spnav` is for client app.

File: spacenav_remote/package.xml (hunk 1 above)
Comment (wzin) [inline spacenav_remote/package.xml]: :ok:

File: spacenav_remote/README.md (hunk 2 above)
Comment (wzin) [inline spacenav_remote/README.md]: :+1:

**Pull request merged**